libs/Helpers/src/Helpers/helpers.h

```diff
@@ -11,4 +15,18 @@ constexpr bool isDebugBuild()
   #endif
 }
 
+/// Case insensitive comparision with optional limitation to first \c len characters
+/// \note \c T entry type must support \c tolower
+/// \todo test me
+template<typename T>
+bool iequals(const T& a, const T& b, size_t len = -1)
```

Comment:
Can we use std::ranges from C++20?
```
bool iequals(const T& a, const T& b) {
    auto to_lower{ std::ranges::views::transform(std::tolower) };
    return std::ranges::equal(a | to_lower, b | to_lower);
}
```

Reply:
Already tried it but found that "libstdc++'s <ranges> does not work with clang at all, because clang's support for concepts is incomplete.". Tried this morning with @saledjenic to use GCC instead on macOS and it isn't straightforward to switch.